kolibri/plugins/device/assets/src/views/FacilitiesPage/FacilityTaskPanel.vue

```diff
@@ -25,7 +25,8 @@
     removeFacilityTaskDisplayInfo,
     importFacilityTaskDisplayInfo,
   } from '../syncTaskUtils';
-  import TaskPanel from './TaskPanel';
+  import { TaskTypes } from '../../constants';
+  import TaskPanel from './FacilityTaskPanelDetails';
```

Comment:
For the sake of consistency I would expect
`import FacilityTaskPanelDetails from './FacilityTaskPanelDetails';` (though I am not sure what are Kolibri naming conventions in this regard)